EcoSoapBank/Helpers/SwiftUI+Extensions.swift

```diff
@@ -5,6 +5,7 @@
 //  Created by Jon Bash on 2020-08-07.
 //  Copyright © 2020 Spencer Curtis. All rights reserved.
 //
+// swiftlint:disable shorthand_operator
 
```

Comment:
Why disable the shorthand_operator rule?

Reply:
Added clarification in PR comment.